README.md

```diff
@@ -134,11 +134,23 @@ For Linux it needs to be built from source.
 
 If you update the gRPC API, you need to regenerate the Protobuf file.
 
-You will need to install the following binaries:
+You will need to install the following binaries in your Go path:
 
 - `go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway`
 - `go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger`
 
+If you don't have swagger generator, you will need it for generating the REST API documentation:
+
+`brew install swagger-codegen`
+`brew install statik`
+
+Finally, you will need to install the git submodules for some extra dependencies in the proto buffer generation:
+
+```
+git submodule init
```

Comment:
I was just thinking we copied the `googleapi` directly into our codebase, so we can avoid using submodules or depending on their repository. This way we can be sure that they don't introduce any breaking changes we are not aware of (although, I don't think the protocol buffer definitions will be changing that often) and also no need for submodule inits. Let me know what you think

Reply:
Fixed in 4855f844713150c70853c9596be8494d139a4485